src/data/mv_polynomial/basic.lean

```diff
@@ -147,6 +147,13 @@ lemma C_injective (σ : Type*) (R : Type*) [comm_semiring R] :
   function.injective (C : R → mv_polynomial σ R) :=
 finsupp.single_injective _
 
+lemma C_surjective {R : Type*} [comm_ring R] :
```

Comment:
We have too many ways to talk about empty types... `fin 0` is an obvious one in the context of polynomials, but there is also `empty` and `pempty`. So I wonder if we should abstract over that. Unfortunately, we don't have a typeclass for empty types, so the best we have is `\not nonempty X`.

Reply:
Created the more generic `\not nonempty X` version and then renamed this to `C_surjective_fin_0`